docs/developer/extending/apps/apps-patterns/persistence-with-metadata-manager.mdx

```diff
@@ -1,3 +1,111 @@
 ---
 title: App Pattern - Persistence with MetadataManager
 ---
+
+## Why would you want a metadata manager?
+
+Apps usually have configuration data that can be changed by the user during runtime:
+
+- access keys to the external APIs (ex. key to a payment gateway)
+- preferences like how the app should look (ex. customized messages)
+- optional features which can be turned off or on (ex. if an additional notification should be sent to the user)
+
+Depending on the type of app you are creating, you might need a dedicated database for such data, but in most of cases, a simpler solution will be perfectly fine.
+Such a solution available out of the box is PrivateMetadata.
+
+## Whats Metadata and its features
+
+[Multiple of objects](developer/api-reference/interfaces/object-with-metadata.mdx) in the Saleor API has a way to add additional Metadata for the objects. In this document, we'll focus on App Metadata.
```

Comment:
Many objects?

Reply:
"Multiple of objects ... for the objects" - I think entire sentence can be rewritten

Reply:
Rephrased the sentence. Is it better now?